### ISSUE TITLE
Action items related to permissions management UI  

### ISSUE THREAD

**Issue by ksens** [Paradigm4/revealgenomics-ui-modules#2]
@arko-bhattacharya recording our discussion here for easier tracking (you can check off items that get completed)

- [x] 1. loop one user at a time while granting permissions
- [x] 2. description
	- has to be filled when "Request access"
	- goes blank when user clicks "Request access"
	- unique [user, study] should be stored + [timestamp, description] to be updated to recent most
- [x] 3. refactor all backend permissions DB related code into functions in a separate file
- [x] 4. timing of when to record to permisssions DB (only after "grant permissions" is succesful)
- [ ] 5. CRON job 
	- see https://github.com/Paradigm4/revealgenomics-ui-modules/issues/1#issuecomment-488333845
- [x] 6. [Lower priority] splitting `status_time` column into three columns

**Comment by ksens** [comment]
@arko-bhattacharya 

Forgot to mention during our TCon yesterday -- could you please plan to commit as much as you have completed by Sunday May 12 EOD into this repo. 

I will then deploy your work onto [customer A](https://github.com/Paradigm4/cs-private-notes-space/blob/master/customer-code-names/revealgenomics-customers.md)'s machine and demo the current status to customer A at  our weekly meeting with them (held every Monday). 

You can then continue with remaining items till we have our next 1x1.

**Comment by ksens** [comment]
3 and 6 completed by @arko-bhattacharya

**Comment by ksens** [comment]
Moved item 5 on list to issue #5.
